### PR TITLE
[Luna-1372][BpkAccordion] Remove description filter

### PR DESCRIPTION
**Pull request by juanmataruiz**
### Description
🎫 [Luna-1371](https://skyscanner.atlassian.net/browse/LUNA-1371) for reference.

There is an issue on iOS where VoiceOver reads the word "description" as it goes through the flight's filter Accordion.

We have identified that this is due to the usage of `dt` `dl` and `dd` in the component as VioceOver will read the element attached to it.

### Checklist

Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [x] `README.md` (If you have created a new component)
- [x] Component `README.md`
- [x] Tests
- [x] Accessibility tests
    - The following checks were performed:
        - [x] Ability to navigate using a [keyboard only](https://webaim.org/techniques/keyboard/)
        - [x] Zoom functionality ([Deque University explanation](https://dequeuniversity.com/checklists/web/text)):
            - [x] The page SHOULD be functional AND readable when only the text is magnified to 200% of its initial size
            - [x] Pages must reflow as zoom increases up to 400% so that content continues to be presented in only one column i.e. Content MUST NOT require scrolling in two directions (both vertically and horizontally)
        - [x] Ability to navigate using a [screen reader only](https://webaim.org/articles/screenreader_testing/)
- [x] Storybook examples created/updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -180,5 +180,8 @@
     "webpack": "^4.30.0",
     "webpack-cli": "^3.3.2",
     "webpack-dev-server": "^3.3.1"
+  },
+  "dependencies": {
+    "@skyscanner/backpack-web": "^34.10.1"
```
Comment (olliecurtis) [inline package.json:185]: This isn't to have been installed can you remove?

File: packages/bpk-component-accordion/src/BpkAccordion.js [hunk 2]
```diff
@@ -35,9 +35,9 @@ const BpkAccordion = (props: Props) => {
 
   return (
     // $FlowFixMe[cannot-spread-inexact] - inexact rest. See decisions/flowfixme.md
-    <dl className={classNames} {...rest}>
+    <div className={classNames} {...rest}>
```
Comment (olliecurtis) [inline packages/bpk-component-accordion/src/BpkAccordion.js:38]: Let me do some checking here for this change, as I think some of these were made for SEO purposes so we may need to do some logic to support both

File: package.json (hunk 1 above)
Comment (juanmataruiz) [reply package.json]: Oh, I missed this. Done now thanks!

File: packages/bpk-component-accordion/src/BpkAccordion.js (hunk 2 above)
Comment (juanmataruiz) [reply packages/bpk-component-accordion/src/BpkAccordion.js:38]: Thanks for doing this. 

If the description elements can't be removed, I would need to chat with the backpack team about the best approach to fix this bug.

**Pull request closed without merging**